platform/android/java/app/config.gradle

```diff
@@ -1,11 +1,11 @@
 ext.versions = [
-    androidGradlePlugin: '3.5.3',
+    androidGradlePlugin: '4.1.0',
     compileSdk         : 29,
     minSdk             : 18,
     targetSdk          : 29,
-    buildTools         : '29.0.3',
+    buildTools         : '30.0.1',
```

Comment:
I'm not super knowledgeable on how `buildTools` relate to the target SDK, but do we want to bump `compileSdk` and `targetSdk` to 30 eventually too?

And that's likely something we'll want to do in `3.2` too as usually Google makes this a requirement for new apps at some point. Judging by previous years, that would be from ~August 2021 (not announced yet): https://support.google.com/googleplay/android-developer/answer/113469#targetsdk

Reply:
We need to resolve https://github.com/godotengine/godot/issues/38913 before we can bump the `targetSdk` and `compileSdk` as the [current workaround](https://developer.android.com/about/versions/11/privacy/storage#scoped-storage) we're using is ignored when targeting Android 11 (API level 30).